presto-main/src/main/java/io/prestosql/type/UuidOperators.java

```diff
@@ -60,6 +61,15 @@ public static Slice uuid()
         return wrappedLongArray(uuid.getMostSignificantBits(), uuid.getLeastSignificantBits());
     }
 
+    @Description("Generates a deterministic (type 3) UUID using the specified input name")
+    @ScalarFunction(deterministic = true)
+    @SqlType(StandardTypes.UUID)
+    public static Slice uuid3(@SqlType(StandardTypes.VARCHAR) Slice name)
+    {
+        java.util.UUID uuid = nameUUIDFromBytes(name.byteArray());
```

Comment:
```suggestion
        java.util.UUID uuid = nameUUIDFromBytes(name.getBytes());
```

Can we use this instead ?

Reply:
> Can we use this instead ?

Sure, but may please I know the reason for this? I see both the methods in the `Slice` class. Just curious as to why one over the other.

Reply:
`Slice#byteArray` returns the `byte[]` wrapped by this Slice and not all instances of Slice are backed by the byte[]. 
Plus if we have Slice which backed by `byte[]` and if we try to get a sub region of it using `Slice#slice(index, length)` both the Slice object share the same `byte[]`. So it is better to use `Slice#getBytes`